### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 36

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#212] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255038-a9b4b108-a3ff-4b0a-9c5b-fbbe233d0cbb.png)

**Comment by i0nut02** [comment]
1.  14! / (2!*2!*2!*2!)
2. 14! / (2!*2!*2!*2!) - 13! / (2!*2!*2!)
3. (12! / (2!*2!)) * 2   - 10! / (2!*2!)
